### PR TITLE
Minor changes to Windows install article

### LINKED ISSUE #42000: Feedback: MSI is mentioned but it's a Windows Installer executable
### Type of issue

Typo

### Description

> I found what appears to be a minor error. If you search the page for the one occurrence of "MSI", it indicates that it will download the MSI version installer, however, on the files I tried, it was the EXE version installer. For my purposes, I would prefer the MSI version. If the EXE is just a wrapper for the MSI, then it shouldn't be a problem to extract the MSI, just extra work.

### Page URL

https://learn.microsoft.com/en-us/dotnet/core/install/windows

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/core/install/windows.md

### Document Version Independent Id

d11d0d14-8a03-50e4-5c00-9a4417ad4d0a

### Article author

@adegeo

### Metadata

* ID: 1eeab451-820f-de42-983e-4d0d5764f6ab 
* Service: **dotnet-fundamentals**
* Sub-service: **install**

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Rework intro
- Remove .msi file reference
- Simplify a little bit about VS

Fixes #42000 

@BillWagner 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/install/windows.md](https://github.com/dotnet/docs/blob/e9b2a470c94157c5cffb7b8a4003930e114ef511/docs/core/install/windows.md) | [docs/core/install/windows](https://review.learn.microsoft.com/en-us/dotnet/core/install/windows?branch=pr-en-us-43022) |

### CODE REVIEW

File: docs/core/install/windows.md [hunk 1]
```diff
@@ -16,30 +16,30 @@ ms.date: 05/15/2024
...
 | **.NET Runtime**         | Yes                           | No                            | No                            |
 | **.NET Desktop Runtime** | Yes                           | Yes                           | No                            |
 | **ASP.NET Core Runtime** | No                            | No                            | Yes                           |
 | **.NET SDK**             | Yes                           | Yes                           | Yes                           |
 
-To ensure that you can run all .NET apps on Windows, install both the ASP.NET Core Runtime and the .NET Desktop Runtime. The ASP.NET Core Runtime runs any web apps, and the .NET Desktop Runtime runs any desktop app, such as a Windows Presentation Foundation (WPF) or Windows Forms (WinForms) app.
+To ensure that you can run all .NET apps on Windows, install both the ASP.NET Core Runtime and the .NET Desktop Runtime. The ASP.NET Core Runtime runs web-based apps, and the .NET Desktop Runtime runs desktop apps, such as a Windows Presentation Foundation (WPF) or Windows Forms (WinForms) app.
```
Comment (gewarren) [inline docs/core/install/windows.md:38]:
```suggestion
To ensure that you can run all .NET apps on Windows, install both the ASP.NET Core Runtime and the .NET Desktop Runtime. The ASP.NET Core Runtime runs web-based apps, and the .NET Desktop Runtime runs desktop apps, such as a Windows Presentation Foundation (WPF) or Windows Forms app.
```

File: docs/core/install/windows.md [hunk 2]
```diff
@@ -16,30 +16,30 @@ ms.date: 05/15/2024
 > - [Install on macOS](macos.md)
 > - [Install on Linux](linux.md)
 
-There are many different ways to install .NET on Windows. This article helps you understand the difference between the SDK and Runtime, which runtime you should install, and the method you should use to install .NET.
+This article teaches you about which versions of .NET are supported on Windows, how to install .NET, and what the difference is between the SDK and Runtime.
 
-.NET is made up of the runtime and the SDK. The runtime runs .NET apps, and the SDK is used to create apps.
+Unlike .NET Framework, .NET isn't tied to your version of Windows. You can only have a single version of .NET Framework installed on Windows. But .NET is standalone and can be installed anywhere on your computer and some apps might include their own copy of .NET.
```
Comment (gewarren) [inline docs/core/install/windows.md:21]:
```suggestion
Unlike .NET Framework, .NET isn't tied to your version of Windows. You can only have a single version of .NET Framework installed on Windows. But .NET is standalone and can be installed anywhere on your computer. Some apps might include their own copy of .NET.
```

File: docs/core/install/windows.md [hunk 3]
```diff
@@ -16,30 +16,30 @@ ms.date: 05/15/2024
 > - [Install on macOS](macos.md)
 > - [Install on Linux](linux.md)
 
-There are many different ways to install .NET on Windows. This article helps you understand the difference between the SDK and Runtime, which runtime you should install, and the method you should use to install .NET.
+This article teaches you about which versions of .NET are supported on Windows, how to install .NET, and what the difference is between the SDK and Runtime.
```
Comment (gewarren) [inline docs/core/install/windows.md:19]:
```suggestion
This article teaches you about which versions of .NET are supported on Windows, how to install .NET, and what the difference is between the SDK and runtime.
```

**Review by gewarren — APPROVED** [review]

**Comment by adegeo** [comment]
Thanks @gewarren

**Pull request merged**